sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 12.9.0-beta.3 (Unreleased)
+## 12.9.0 (2023-09-12)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Features in `12.9.0-beta.1` and `12.9.0-beta.2` are now generally available.
```

Comment:
We need to keep the heading structure in the changelog for our release notes tooling.

In this case, I recommend keeping this bullet under `### Features Added`

Reply:
cc @antkmsft (fyi)

Reply:
BTW, in our releases for other libraries, instead of saying "Changes from beta are now generally available", we actually duplicate the entries from the beta releases, sometimes pulling only the high-level ones and putting less details. This way, customers who only consume our stable releases, are seeing meaningful changelogs.
Otherwise, given that most of the changes go through a beta release first, all changelogs for the stable releases are forever going to look like "changes from <previous beta> are now generally available".